### ISSUE TITLE
Further problems in Windows executables.

### ISSUE THREAD

**Issue by noraesae** [noraesae/koko#13]
The Windows support is added in `v0.2.0`. However, I personally don't own any Windows and that's why it took so long to add it as I should work on a PC of another.

So, when there is a Windows-specific bug, it may take super long for me to fix it by myself. However, fortunately, the bug may be related to Electron because the app source itself is used in the same manner in both OS X and Windows. It means we may not need to look into complicated app source code.

In conclusion, I will really really appreciate if everybody can help me fix further Windows problems. Surely I'll also help as much as I can.

Thanks!

**Comment by RangerMauve** [comment]
Is there a list of known bugs that could be tracked in this issue? I'm going to try installing it later this week so I'll report anything I can find. :D

**Comment by noraesae** [comment]
None has been reported, but there may be at least some. Thanks for your help :smile:
